README.md

```diff
@@ -3,3 +3,19 @@
 [![Build Status](https://travis-ci.org/exercism/arm64-assembly.svg?branch=master)](https://travis-ci.org/exercism/arm64-assembly)
 
 Exercism exercises in ARM64 Assembly.
+
+## Assembler
+
+GNU Assembler (gas) is used for this track.
+
+## Testing
+
+Unity is a simple unit testing framework for C used by this track. Using a framework written in C instead of assembly language is easier to understand. It also shows how parameters are passed when calling assembly routines from C.
```

Comment:
```suggestion
Unity is a simple unit testing framework for C used by this track.
Using a framework written in C instead of assembly language is easier to understand.
It also shows how parameters are passed when calling assembly routines from C.
```